README.md

```diff
@@ -68,9 +68,9 @@ Find more install options on [Apple Silicon](https://docs.jina.ai/get-started/in
 
 ### Basic Concepts
 
-Jina has four fundamental concepts:
+Jina has five fundamental concepts:
```

Comment:
The number of bullets needs to match the number of concepts, otherwise, it looks like we don't know how to count. Either three or four, but the bullets must match the number!

Reply:
But the number of concepts are 5, can you suggest a way of expressing them?

Reply:
I count four: BaseDoc, Executor, Deployment, Flow.